Packs/Cylance_Protect/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+##### Cylance Protect v2
+- **New Commands**  
+Added support for Optics API with 3 new commands:
+- cylance-optics-create-instaquery: create new InstaQuery to search for artifacts in one or multiple zones.
+- cylance-optics-get-instaquery-result: get InstaQuery result
+- cylance-optics-list-instaquery: list all current InstaQueries
+- **Get Policy Details**   
+Added the full context and raw_response result for policy details
```

Comment:
Where was this added? I haven't seen such an addition.

Reply:
Yes, it is pushed to get_policy_details() function.

Reply:
I see it now, it was added after I reviewed to code.